blog/_posts/2023-09-11-10.2-change-collection.md

```diff
@@ -60,6 +69,99 @@ There's a [dedicated post](https://stormearthandlava.com/blog/2023/09/10/t31-fir
+--- | --- | ---
+1 | 10% | 24%
+2 | 20% | 28%
+3 | 30% | 32%
+4 | 40% | 36%
+5 | 50% | 40%
+6 | 60% | 44%
+
+</div>
+  </div>
+</div>
+
+
+
+<br>
+
+## Further Beyond
+
+- {{ site.data.talent.fb }} can now only increase the duration of {{ site.data.talent.fb }} by up to 150% of the original duration of {{ site.data.talent.asc }}.
+
+> Source: [Blue Post](https://us.forums.blizzard.com/en/wow/t/guardians-of-the-dream-ptr-development-notes/1666552/6) (2023-09-27)
+
+<div class="accordion mb-4" id="further-beyond">
+  <div class="card">
+    <div class="card-header" id="fb">
+      <div class="h2 mb-0">
+        <button class="btn btn-link btn-block text-left collapsed" type="button" data-toggle="collapse" data-target="#fb-body" aria-expanded="false" aria-controls="fb-body">
+          General notes
+        </button>
+      </div>
+    </div>
+    <div id="fb-body" class="collapse" aria-labelledby="fb">
+      <div class="card-body" markdown="1">
+
+- From early testing, this seems to double {{ site.data.talent.asc }}'s uptime for build that pick {{ site.data.talent.fb }}, compared to builds who do not, meaning ~45% uptime on it.
+
+</div>
+    </div>
+  </div>
+  <div class="card">
+    <div class="card-header" id="fb-issues">
+      <div class="h2 mb-0">
+        <button class="btn btn-link btn-block text-left collapsed" type="button" data-toggle="collapse" data-target="#fb-issues-body" aria-expanded="false" aria-controls="fb-issues-body">
+          Known Issues
+        </button>
+      </div>
+    </div>
+    <div id="fb-issues-body" class="collapse" aria-labelledby="fb-issues">
+      <div class="card-body" markdown="1">
+
+- There's currently an issue with {{ site.data.talent.dre }} proccing not adding to the {{ site.data.talent.fb }} cap.
+  - When {{ site.data.talent.dre }} procs, your {{ site.data.talent.fb }} cap is 9 seconds (6*1.5), meaning you can go up to 15 seconds of {{ site.data.talent.asc }}.
+  - If you proc {{ site.data.talent.dre }} again while {{ site.data.talent.asc }} is still active, the {{ site.data.talent.fb }} cap will stay at 9s, instead of going up to 18s. You will have 21 seconds of {{ site.data.talent.asc }} max instead of 30.
+- The {{ site.data.talent.further_beyond }} cap doesn't go up to the maximum when overcapping duration.
+  - If you proc {{ site.data.talent.dre }}, the cap will allow you to gain 9 seconds via {{ site.data.talent.fb }}.
+  - Cast {{ site.data.talent.eb }} twice, gain 7s of {{ site.data.talent.asc }}, no problem
+  - Cast {{ site.data.talent.eb }} once more, 7+3.5 = 10.5s which is over the cap set by {{ site.data.talent.fb }}, and you do not gain any extension instead of going up to the cap and gaining 2s of {{ site.data.talent.asc }}.
+
+</div>
+    </div>
+  </div>
+</div>
+
+<br>
+
+## Primordial Wave
+
+- {{ site.data.talent.primordial_wave }} can no longer be used to heal a friendly ally, it is now only an offensive spell for Elemental Shaman.
```

Comment:
I would possibly slightly extend on this?
"This means that you will not be able to accidentally target an ally with the spell, effectively wasting the spell."